board/safety/safety_toyota.h

```diff
@@ -200,6 +201,11 @@ static void toyota_rx_hook(const CANPacket_t *to_push) {
       brake_pressed = GET_BIT(to_push, bit) != 0U;
     }
 
+    // brake pressed on 0x101 for rav4 prime
+    if (addr == 0x101) {
```

Comment:
Do we need another `toyota_alt_brake` flag? The message is not present on other Toyotas.